themes/classic/_dev/js/cart.js

```diff
@@ -168,6 +168,36 @@ $(document).ready(() => {
     });
   };
 
+  //Hide 'have a code promo' text if displayed when 'close' is
+  let isPresent = $('.promo-code-button').length,
+      nameLength = 0;
+
+  function labelCodeDisplayed() {
+    $('.promo-code-button').not('.cancel-promo').hide();
+    
+    $('.cancel-promo').on('click', () => {
+      $('.cancel-promo').hide();
+      $('.promo-code-button').not('.cancel-promo').show();
+    });
+    $('.promo-code-button').not('.cancel-promo').on('click', () => {
+        $('.promo-code-button').not('.cancel-promo').hide();
```

Comment:
$(this) should be working, isn't it?
```suggestion
        $(this).hide();
```

Reply:
No because of es6 notation `() => {} `

it should be 
```js
$('.promo-code-button').not('.cancel-promo').on('click', function hideCancePromo() {
        $(this).hide();
}):
```

Reply:
Interesting. Learnt something one more time.

Reply:
Yes, with `() => {}` `this  `context came from parent / class instead of `function` which have is own `this` context 😄